docs/resource-policy/installation.md

```diff
@@ -41,38 +41,44 @@ following components: DaemonSet, ConfigMap, CustomResourceDefinition, and RBAC-r
           For instance,
 
           ```sh
-          helm install topology-aware --namespace kube-system --set nri.patchRuntimeConfig=true deployment/helm/topology-aware/
+          helm install nri-plugins/nri-resource-policy-topology-aware --namespace kube-system --set nri.patchRuntimeConfig=true
           ```
 
 - Kubernetes 1.24+
 - Helm 3.0.0+
 
 ## Installing the Helm Chart
 
-1. Clone the project to your local machine
+1. Add the nri-plugins charts repository so that Helm install can find the actual charts.
+
     ```sh
-    git clone https://github.com/containers/nri-plugins.git
+    helm repo add nri-plugins https://containers.github.io/nri-plugins
     ```
 
-1. Navigate to the project directory
+1. List chart repositories to ensure that nri-plugins repo is added.
+
     ```sh
-    cd nri-plugins
+    helm repo list
     ```
 
-1. Install the plugin using Helm. Replace release name with the desired name
-   for your Helm release. In this example, we named it as topology-aware. The
-   default values for topology-aware resource policy plugin are stored in
-   values.yaml file. If you wish to provide custom values to the Helm
-   chart, refer to the [table](#helm-parameters) below, which describes the
-   available parameters that can be modified before installation. It's important
-   to note that specifying the namespace (using `--namespace`) is crucial when
-   installing the Helm chart. If no namespace is specified, the manifests will
-   be installed in the default namespace.
+1. Install the plugin. Replace release version with the desired version. If you wish to
+   provide custom values to the Helm chart, refer to the [table](#helm-parameters) below,
+   which describes the available parameters that can be modified before installation.
+   It's important to note that specifying the namespace (using `--namespace` or `-n`) is
+   crucial when installing the Helm chart. If no namespace is specified, the manifests
+   will be installed in the default namespace.
 
     ```sh
-    helm install topology-aware --namespace kube-system deployment/helm/topology-aware/
+    helm install nri-plugins/nri-resource-policy-topology-aware --namespace kube-system --version v0.2.0
```

Comment:
Ach, I accidentally commented #148 first. But the same holds here. Now looking at this, we could also just drop the `--version` parameter (especially in the main branch) to always just install the latest version

```suggestion
    helm install topology-aware nri-plugins/nri-resource-policy-topology-aware --namespace kube-system
```